src/Datasource/EntityTrait.php

```diff
@@ -482,7 +482,7 @@ public function setVirtual(array $properties, $merge = false)
             return $this;
         }
 
-        $this->_virtual += $properties;
+        $this->_virtual = array_merge($this->_virtual, $properties);
```

Comment:
The one @saeideng was talking below thinking it was duplicating things.

Calling `$entity->setHidden(['name'], true);` multiple times keeps the array value once in the array, thus this seems to proof that this does not need array_unique() here.